source/docs/networking/networking-introduction/measuring-bandwidth-usage.rst

```diff
@@ -3,12 +3,12 @@
 Measuring Bandwidth Usage
 =========================
 
-On the FRC\ |reg| Field (and at home when the radio is configured using the FRC Bridge Configuration Utility) each team is limited to 4Mb/s of network traffic (see the `FMS Whitepaper <https://wpilib.screenstepslive.com/s/fms/m/whitepaper/l/608744-fms-whitepaper>`__ for more details). The FMS Whitepaper provides information on determining the bandwidth usage of the Axis camera, but some teams may wish to measure their overall bandwidth consumption. This document details how to make that measurement.
+On the FRC\ |reg| Field (and at home when the radio is configured using the FRC Bridge Configuration Utility) each team is allocated limited (see `R60B <https://firstfrc.blob.core.windows.net/frc2021/Manual/HTML/2021FRCGameManual.htm#R60>`__) network bandwidth (see the `FMS Whitepaper <https://wpilib.screenstepslive.com/s/fms/m/whitepaper/l/608744-fms-whitepaper>`__ for more details). The FMS Whitepaper provides information on determining the bandwidth usage of the Axis camera, but some teams may wish to measure their overall bandwidth consumption. This document details how to make that measurement.
```

Comment:
Reading this again:
```
and at home when the radio is configured using the FRC Bridge Configuration Utility
```

This is only accurate when the bandwidth setting is set.